enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx

```diff
@@ -89,6 +91,20 @@ export const InteractiveQuestionProvider = ({
   );
 };
 
+export const InteractiveQuestionProvider = ({
+  cardId,
+  ...rest
+}: Omit<InteractiveQuestionProviderProps, "options"> & {
+  options?: QueryParams;
+}) => {
+  const id = useValidIdForEntity({
```

Comment:
Is this because we want to preserve `InteractiveQuestionProviderInner` to still work with `cardId` for other usages of this provider?

Reply:
nope, this should work with all question components that use InteractiveQuestionProvider. I didn't want to put it in the actual components since we might be using the provider for other question components in the future and I didn't want to have to put the entity ID logic in all of the places